daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/parsers/SequenceChildBases.scala

```diff
@@ -509,7 +511,7 @@ trait EndArrayChecksMixin {
 
   def erd: ElementRuntimeData
 
-  def endArray(state: ParseOrUnparseState, actualOccurs: Long): Unit = {
+  def endArray(state: ParseOrUnparseState, occurrence: Long): Unit = {
```

Comment:
I'm fine with keeping or renaming `actualOccurs`, but the new name should be plural as well (`occurrences` or `numOccurrences`, which isn't all that different from `actualOccurs` actually).

Reply:
These changes have been replaced with counting the children in the array node.